lib/android-helpers.js

```diff
@@ -335,6 +335,14 @@ helpers.initUnicodeKeyboard = async function (adb) {
   return defaultIME;
 };
 
+helpers.setMockLocationApp = async function (adb, app) {
+  if (await adb.getApiLevel() < 23) {
```

Comment:
I would go ahead and do that right now if this was a mono repo but I believe if we even decide  to change adb.getApiLevel we should expect a string for now and change adb repo after we merge this. With projects in the structure like appium, with a lot of small repos, I like go step by step usually and make sure that the thing I'm changing doesn't rely on a change in a different repo. Even though, as you said, JS does implicit conversion and it probably wouldn't break anything